src/components/Game.tsx

```diff
@@ -0,0 +1,65 @@
+import { type } from "@testing-library/user-event/dist/type";
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game() {
+  const resetbox: Array<string> = Array(9).fill("");
```

Comment:
FYI

you did not clarify the type of a variable; it had already been clarified by assigning the value.